### PR TITLE
Update cilium docs

### PR DESCRIPTION
**Pull request by aanm**
Update Cilium pieces in the documentation with the latest v1.10 release

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 1]
```diff
@@ -24,45 +24,59 @@ For background on Cilium, read the [Introduction to Cilium](https://docs.cilium.
...
+minikube version: v1.5.2
 ```
 
 ```shell
-minikube start --network-plugin=cni --memory=4096
+minikube start --network-plugin=cni
 ```
 
-Mount the BPF filesystem:
+For minikube you can install Cilium using this simple CLI tool. Cilium will
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:44]: nit: Can you remove the word "simple"? Thanks.

File: content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 2]
```diff
@@ -85,14 +99,14 @@ Deploying a cluster with Cilium adds Pods to the `kube-system` namespace. To see
 this list of Pods run:
 
 ```shell
-kubectl get pods --namespace=kube-system
+$ kubectl get pods --namespace=kube-system -l k8s-app=cilium
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:102]: nit: You can remove the command prompt ($). Thanks.

File: content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 3]
```diff
@@ -24,45 +24,59 @@ For background on Cilium, read the [Introduction to Cilium](https://docs.cilium.
 ## Deploying Cilium on Minikube for Basic Testing
 
 To get familiar with Cilium easily you can follow the
-[Cilium Kubernetes Getting Started Guide](https://docs.cilium.io/en/stable/gettingstarted/minikube/)
+[Cilium Kubernetes Getting Started Guide](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default//)
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:27]: nit: Is the second forward slash needed?

**Comment by kbhawkey** [comment]
Hi @aanm .
Can you rebase your changes into a single commit? Thanks.
/sig network

**Review by sftim — COMMENTED** [review]
@aanm, are you able to recommend someone to provide technical review for this PR?

File: content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md (hunk 3 above)
Comment (aanm) [reply content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:27]: No it is not, removed.

**Comment by aanm** [comment]
> @aanm, are you able to recommend someone to provide technical review for this PR?

@sftim You mean someone else besides me? I'm a co-maintainer of Cilium so it is in my best interest the technical side is correct :-)

**Comment by aanm** [comment]
> Hi @aanm .
> Can you rebase your changes into a single commit? Thanks.
> /sig network

@kbhawkey Done, thank you for the review!

**Comment by sftim** [comment]
Hmm. @danwent you're listed as a reviewer but it seems you're not a Kubernetes org member. Is that right?

File: content/en/docs/tasks/network/validate-dual-stack.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clu
 
 
 * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
-* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Kubenet or Calico)
+* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Kubenet, Calico or Cilium)
```
Comment (sftim) [inline content/en/docs/tasks/network/validate-dual-stack.md:19]:
```suggestion
* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Calico, Cilium, or Kubenet)
```
(It's CNCF policy to list 3rd party products alphabetically)

**Pull request merged**